src/drt/src/db/tech/frConstraint.h

```diff
@@ -80,12 +80,152 @@ class frConstraint
+        return "SpacingTable";
+
+      case frConstraintTypeEnum::frcSpacingTableTwConstraint:
+        return "SpacingTableTw";
+
+      case frConstraintTypeEnum::frcLef58SpacingTableConstraint:
+        return "Lef58SpacingTable";
+
+      case frConstraintTypeEnum::frcLef58CutSpacingTableConstraint:
+        return "Lef58CutSpacingTable";
+
+      case frConstraintTypeEnum::frcLef58CutSpacingTablePrlConstraint:
+        return "Lef58CutSpacingTablePrl";
+
+      case frConstraintTypeEnum::frcLef58CutSpacingTableLayerConstraint:
+        return "Lef58CutSpacingTableLayer";
+
+      case frConstraintTypeEnum::frcLef58CutSpacingParallelWithinConstraint:
+        return "Lef58CutSpacingParallelWithin";
+
+      case frConstraintTypeEnum::frcLef58CutSpacingAdjacentCutsConstraint:
+        return "Lef58CutSpacingAdjacentCuts";
+
+      case frConstraintTypeEnum::frcLef58CutSpacingLayerConstraint:
+        return "Lef58CutSpacingLayer";
+
+      case frConstraintTypeEnum::frcMinimumcutConstraint:
+        return "Minimumcut";
+
+      case frConstraintTypeEnum::frcLef58CornerSpacingConcaveCornerConstraint:
+        return "Lef58CornerSpacingConcaveCorner";
+
+      case frConstraintTypeEnum::frcLef58CornerSpacingConvexCornerConstraint:
+        return "Lef58CornerSpacingConvexCorner";
+
+      case frConstraintTypeEnum::frcLef58CornerSpacingSpacingConstraint:
+        return "Lef58CornerSpacingSpacing";
+
+      case frConstraintTypeEnum::frcLef58CornerSpacingSpacing1DConstraint:
+        return "Lef58CornerSpacingSpacing1D";
+
+      case frConstraintTypeEnum::frcLef58CornerSpacingSpacing2DConstraint:
+        return "Lef58CornerSpacingSpacing2D";
+
+      case frConstraintTypeEnum::frcLef58SpacingEndOfLineConstraint:
+        return "Lef58SpacingEndOfLine";
+
+      case frConstraintTypeEnum::frcLef58SpacingEndOfLineWithinConstraint:
+        return "Lef58SpacingEndOfLineWithin";
+
+      case frConstraintTypeEnum::
+          frcLef58SpacingEndOfLineWithinEndToEndConstraint:
+        return "Lef58SpacingEndOfLineWithinEndToEnd";
+
+      case frConstraintTypeEnum::
+          frcLef58SpacingEndOfLineWithinEncloseCutConstraint:
+        return "Lef58SpacingEndOfLineWithinEncloseCut";
+
+      case frConstraintTypeEnum::
+          frcLef58SpacingEndOfLineWithinParallelEdgeConstraint:
+        return "Lef58SpacingEndOfLineWithinParallelEdge";
+
+      case frConstraintTypeEnum::
+          frcLef58SpacingEndOfLineWithinMaxMinLengthConstraint:
+        return "Lef58SpacingEndOfLineWithinMaxMinLength";
+
+      case frConstraintTypeEnum::frcLef58CutClassConstraint:
+        return "Lef58CutClass";
+
+      case frConstraintTypeEnum::frcRecheckConstraint:
+        return "Recheck";
+
+      case frConstraintTypeEnum::frcLef58EolExtensionConstraint:
+        return "Lef58EolExtension";
+
+      case frConstraintTypeEnum::frcLef58EolKeepOutConstraint:
+        return "Lef58EolKeepOut";
+      default:
+        return "UNKNOWN";
```

Comment:
Remove the default case and handle all values (this will also error if we add a new type and forget to update this).